utils.sh

```diff
@@ -47,7 +47,9 @@ function create_cluster() {
     generate_templates
 
     mkdir -p ${assets_dir}/openshift
-    cp -rf assets/generated/*.yaml ${assets_dir}/openshift
+    for f in assets/generated/*.yaml; do
```

Comment:
what about using find instead ?

Reply:
What benefit is there of using find over globbing in this case?

Reply:
Yeah you're right - this approach still suffers from the same shell globbing problem as the `cp`

Reply:
I guess that depends on how many files we expect to copy, find is supposed to be faster than a loop for large amount of files

Reply:
Yeah I think the number of files should be small, but I forgot that globbing will pass the literal pattern into the loop in the event it matches no files, so find is probably cleaner - thanks! :)